src/elements/button/button.js

```diff
@@ -2,7 +2,7 @@
 'use strict';
 
 angular
-  .module('semantic.ui.elements.button', [])
+  .module('semantic.ui.elements.button', ['semantic.ui.elements.icon', 'semantic.ui.elements.divider'])
```

Comment:
your PR is about a new directive - sm-icon. You made changes in another module and are not related to the new feature, if you want to propose some changes for another element (module) it should be in a separated PR.